lib/beaker-hostgenerator/cli.rb

```diff
@@ -66,6 +68,14 @@ def initialize(argv = ARGV.dup)
           @options[:list_supported_values] = true
         end
 
+        opts.on('--extract-templates-from-file HOSTS_FILE') do |f|
+          @options[:extract_templates] = f
+        end
+
+        opts.on('--templates-only') do
+          @options[:templates_only] = true
+        end
```

Comment:
In this fictional world, is there value in doing a roll-up of these, so we end up with:

```
centos-6-x86_64=1,ubuntu-1404-i386=3

```